### PR TITLE
Align form status checkmark consistently

### LINKED ISSUE #146: Notes sync and checkmark icons in different places on same page
The little green check is on the bottom right for the analysis description and the bottom left for analysis details. 
Proposal: both go on bottom right, or at least both in the same relative position.

Screen Shot 2023-02-14 at 5 44 33 AM

### PR DESCRIPTION
**Pull request by jernestmyers**
Resolves #146 

From Storybook:

With a character limit (no change):
image

Without a character limit:
image

### CODE REVIEW

File: src/components/forms/MultilineTextField.tsx [hunk 1]
```diff
@@ -166,7 +166,7 @@ export default function MultilineTextField({
               {`${value.length} / ${characterLimit}`}
             </p>
           )}
-          <div css={{ paddingTop: 3 }}>
+          <div css={{ paddingTop: 3, marginLeft: characterLimit ? undefined : 'auto' }}>
```
Comment (adnauseum) [inline src/components/forms/MultilineTextField.tsx:169]: **question**: Not an experienced Emotion 👨‍🎤 user. does `marginLeft: undefined` boil down into: `margin-left: unset`?

**Review by adnauseum — APPROVED** [review]
Send it! 🚀

File: src/components/forms/MultilineTextField.tsx (hunk 1 above)
Comment (jernestmyers) [reply src/components/forms/MultilineTextField.tsx:169]: I _**think**_ from my past memory of checking this behavior in the dev tools, the styling just gets ignored when set to undefined. If memory serves me, the inline styling would show as `padding-top: 3; margin-left: auto` if there is not a character limit and `padding-top: 3` if there is a character limit.

Comment (adnauseum) [reply src/components/forms/MultilineTextField.tsx:169]: 👍 🙇

**Comment by asizemore** [comment]
yikes i read through the changes and just never gave a review. Sorry @jernestmyers but thanks @adnauseum for picking up my slack!

**Comment by adnauseum** [comment]
> yikes i read through the changes and just never gave a review. Sorry @jernestmyers but thanks @adnauseum for picking up my slack!


My first thought: _Sheesh, Ann's **really slacking**_. Once again—what would they do without me? 🤔 💭

**Pull request merged**